src/Alert/Alert.scss

```diff
@@ -37,6 +37,11 @@
   display: flex;
 }
 
+// adds starting margin to align text with heading
+p {
+  margin-inline-start: $alert-icon-width + $alert-icon-space
+}
```

Comment:
This is a brittle way to achieve the desired result:

- If someone uses a `p` tag but not an icon then the text will be indented for no reason
- If someone uses a `ul` tag for content then it will not be indented the same as a paragraph.

Let's add our own shim for this react-bootstrap component and do two things:
- Add an `icon` prop to render the icon
- Put all children inside an `alert-content` div on which you can either add padding or use flexbox on the parent container to layout the icon + content.

See https://gist.github.com/abutterworth/8ead64d686523c804fceed61a182146c for an example.